### PR TITLE
Add ElementType ACCORDION, numberOfDecimals, update RespondentGroup to include Helsenorge

### PR DESCRIPTION
**Pull request by janmlam**
https://github.uio.no/IT-NETTSKJEMA/nettskjema-api/pull/4503

### CODE REVIEW

File: schema/2023-03/Form.json [hunk 1]
```diff
@@ -325,7 +330,8 @@
         "SELECT",
         "SUBMISSION_REFERENCE",
         "TEXT",
-        "USERNAME"
+        "USERNAME",
+        "ACCORDION"
```
Comment (f-krull) [inline schema/2023-03/Form.json]: enum is sorted alphabetically
move to top?

Comment (janmlam) [reply schema/2023-03/Form.json]: Oh, yea thanks ill move it

**Review by kouylekov-usit — DISMISSED** [review]

**Review by kajh — DISMISSED** [review]

**Comment by f-krull** [comment]
@janmlam 
* Adding the accordion might be a breaking change. Should we increase the version?
* (not directly related to this PR:) the `version` field is missing in the schema. It's already being sent (currently with value `2023-03`). We should add it to the schema.
* (not directly related to this PR:) How about a second field `version_minor`? We can use it for non-breaking changes. We had many of those in the past

**Comment by janmlam** [comment]
> @janmlam
> 
> * Adding the accordion might be a breaking change. Should we increase the version?
> * (not directly related to this PR:) the `version` field is missing in the schema. It's already being sent (currently with value `2023-03`). We should add it to the schema.
> * (not directly related to this PR:) How about a second field `version_minor`? We can use it for non-breaking changes. We had many of those in the past

Yea, I'm open to increase the version. Do I make a duplicate of `2023-03` to `2024-09`, then make the changes there?

**Comment by f-krull** [comment]
> Yea, I'm open to increase the version. Do I make a duplicate of `2023-03` to `2024-09`, then make the changes there?

I think that would make sense. @uio-torjus ?

File: schema/2024-09/Form.json [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+      "type": "string",
+      "enum": [
+        "ALL",
+        "FEIDE",
+        "INVITED",
+        "ID_PORTEN_LEVEL_3",
+        "ID_PORTEN_LEVEL_3_SIGNING",
+        "ID_PORTEN_LEVEL_4",
+        "ID_PORTEN_LEVEL_4_SIGNING",
+        "HELSENORGE"
```
Comment (janmlam) [inline schema/2024-09/Form.json:128]: This is the change

File: schema/2024-09/Form.json [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+          "type": "string"
+        }
+      }
+    },
+    "elementType": {
+      "description": "Nettskjema question types.",
+      "$comment": "https://github.uio.no/IT-NETTSKJEMA/nettskjema-api/blob/master/src/main/java/no/uio/webapps/nettskjema/api/v3/form/enums/ElementType.java",
+      "type": "string",
+      "enum": [
+        "ACCORDION",
```
Comment (janmlam) [inline schema/2024-09/Form.json:313]: And this is the change

File: schema/2024-09/Form.json [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+        "answerOptionsVisibilityFilter": {
+          "description": "List of answerOptionIDs. This element becomes visible if one of the options is selected.",
+          "type": "array",
+          "items": [{ "$ref": "#/$defs/visibilityFilterItem" }],
+          "minItems": 1
+        },
+        "numberOfDecimals": {
+          "description": "How many decimals if it is a number element.",
+          "type": "number"
+        }
```
Comment (janmlam) [inline schema/2024-09/Form.json:244]: This is also a change

**Comment by uio-torjus** [comment]
> > Yea, I'm open to increase the version. Do I make a duplicate of `2023-03` to `2024-09`, then make the changes there?
> 
> I think that would make sense. @uio-torjus ?

:+1:

I just need a heads up before nettskjema actually starts delivering with new version, as it needs to be added to the loader.

It's a tiny change, so will be quick to add.

**Review by uio-torjus — APPROVED** [review]

**Review by f-krull — APPROVED** [review]

**Pull request merged**